.azure-pipelines_ci_test.yml

```diff
@@ -0,0 +1,34 @@
+# Python package
```

Comment:
for CI: run unit_test AND pre_trained model with 1 fixed test setup.
2 yml files.

for nightly: run unit_test AND pre_trained model with all test matrix.
2 yml files.

Reply:
if there are too many azure pipeline files in root dir, u may consider put them to a separate folder.